test/mocks/components/remaining-amount.js

```diff
@@ -1 +1,11 @@
-module.exports = 200.01
+const topUpRemainingAmount = 200.01

```

Comment:
These values aren't just thought up to match 'must be a float' and for recovery 'must be negative'.

These are meant to collate to real test scenarios given in our int tests, by providing top-up from `mock-schedule` results in a remaining amount of this, whereas providing reduction from `mock-schedule` results in a remaining amount of line 2 because the rest of the data and the processing of the system output tht.

This is a smell that valuable integration tests haven't been done.

Reply:
There is separation of responsibility between statement-constructor and statement-generator. statement-constructor deals with calculation and correctness of data manipulation, while statement-generator deal with validating and checking data-value from statement-constructor, then generates documents based of data-values that are received from statement-constructor.

Reply:
Yeee understand that -- I just want our mock test values to mimic as close as possible a real top-up schedule or reduction-schedule, etc

Which because we just pick up messages from a Topic isn't difficult to get.